include/pybind11/detail/class.h

```diff
@@ -24,6 +24,14 @@ PYBIND11_NAMESPACE_BEGIN(detail)
 #  define PYBIND11_SET_OLDPY_QUALNAME(obj, nameobj) setattr((PyObject *) obj, "__qualname__", nameobj)
 #endif
 
+inline std::string get_tp_name(PyTypeObject *type) {
+#if !defined(PYPY_VERSION)
```

Comment:
What happens in 7.3.2? Does this line start working?

Reply:
Ugh .. I'd written this yesterday, but it was still a "pending review" ...